README.md

```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.org/perl6/doc.svg?branch=master)](https://travis-ci.org/perl6/doc) [![artistic](https://img.shields.io/badge/license-Artistic%202.0-blue.svg?style=flat)](https://opensource.org/licenses/Artistic-2.0) [![Run Status](https://api.shippable.com/projects/591e99923f2f790700098a30/badge?branch=master)](https://app.shippable.com/github/perl6/doc)
 
 An HTML version of this documentation can be found at [https://docs.perl6.org/](https://docs.perl6.org/).
-This is currently the recommended way to consume the documentation.
+This is currently the recommended way to consume the documentation. 
```

Comment:
Uh, oh, whitespace added here. Can you please check?

Reply:
the build blocked because it could not find Pod::To::HTML v0.6.1  
The problem - I think - because I had only just merged the patch.  
zef now recognises 0.6.1  
I didn't know another way to get travis to rebuild :(

Reply:
Travis build now recognises the Pod::To::HTML v 0.6.1